piksi_tools/console/settings_view.py

```diff
@@ -212,6 +227,40 @@ def _factory_default_button_fired(self):
                         + "Are you sure you want to reset to factory defaults?"
     confirm_prompt.run(block=False)
 
+  def _settings_save_to_file_fired(self):
+    dialog = FileDialog(label='Choose Location to save settings file',
+                        action='save as', wildcard=self.file_wildcard)
+    dialog.open()
+    if dialog.return_code == OK:
+      filepath = os.path.join(dialog.directory, dialog.filename)
+      self.save_settings_to_file(filepath)
+    else:
+      print 'Unable to save file'
+
+  def save_settings_to_file(self, filepath):
+    fio = FileIO(self.link)
+    with open(filepath, 'w') as config_file:
+      blob = fio.read("config")
+      config_file.write(blob)
```

Comment:
Is it going to be obvious that you're savings the current Piksi settings to a file, not the current set of settings in the console (that may or may not have been saved to Piksi's flash)?